com.prateekmedia.appimagepool.yaml

```diff
@@ -0,0 +1,50 @@
+app-id: com.prateekmedia.appimagepool
+runtime: org.gnome.Platform
+runtime-version: "40"
+sdk: org.gnome.Sdk
+command: appimagepool
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
+  - --device=all
+  - --filesystem=home
+  - --filesystem=xdg-download
```

Comment:
do you really need home permissions? it also includes `xdg-download`.

Reply:
I need permission to /home/user/Applications folder (as this is standard path for appimage's) is their any way to specify that or get this path.

Reply:
how about `--persist=Applications`?